.golangci.yml

```diff
@@ -0,0 +1,85 @@
+# v1.47.2
+# Please don't remove the first line. It uses in CI to determine the golangci version
```

Comment:
We should either remove that line since we don't use it in the CI, or start in CI using, as k6 does:
https://github.com/grafana/k6/blob/master/.github/workflows/lint.yml#L48-L57

Reply:
Oh, thank you! When I brought this over, didn't realize the job was not actually looking at this comment. Adding parse logic to the job.